src/servers/realtime/Dockerfile

```diff
@@ -12,14 +12,14 @@ RUN yarn build
 RUN rm -rf /app/node_modules
 RUN yarn install --frozen-lockfile --production
 
-FROM gcr.io/distroless/nodejs:16
+FROM node:16-alpine
```

Comment:
should move this docker file too...

Probably should be under `images/{realtime,history}/Dockerfile`